@navikt/core/react/src/overlays/action-menu/ActionMenu.tsx

```diff
@@ -349,38 +350,14 @@ const ActionMenuTrigger = forwardRef<HTMLButtonElement, ActionMenuTriggerProps>(
               }
             }
           })}
-          /* TODO: Use onClick instead of these two */
+          onClick={composeEventHandlers(onClick, () => context.onOpenToggle())}
```

Comment:
Do we need to check `disabled` here as well?

Reply:
No since onClick cant be triggered on disabled item, i think we can skip the check

Reply:
Then we can do this:
```suggestion
          onClick={composeEventHandlers(onClick, context.onOpenToggle)}
```

Reply:
Fixed 🚀